metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/AlertServiceImpl.java

```diff
@@ -37,15 +47,21 @@
 @Service
 public class AlertServiceImpl implements AlertService {
 
```

Comment:
can we make the type metron_alert?  alert seems a little generic

Reply:
Sure I can do that.  Since these settings are specific to the Alerts UI, should we include "ui" too?  Should the REST service also follow a similar pattern?

Reply:
That seems reasonable

Reply:
Name is changed in the latest commit.  Let me know what you think.

Reply:
I think that is great.  At some point, if we are going to do this a few times we should have a standard naming convention ( separators at least or something).  But that isn't a thing for this PR